config/default.json

```diff
@@ -8,5 +8,6 @@
 		"CrossrefREST.email": "" // Pass an email to Crossref REST API to utilize the faster servers pool
 	},
 	"userAgent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10.13; rv:61.0) Gecko/20100101 Firefox/61.0",
-	"translatorsDirectory": "./modules/translators"
+	"translatorsDirectory": "./modules/translators",
+	"persistentCookies": false
```

Comment:
Let's add a comment here to make it clear what this does, since it's a bit of a privacy risk:

`"persistentCookies": false // Remember upstream cookies between translation operations. Do not enable for a public service.`

Reply:
done!